wiki/Ranking_Criteria/id.md

```diff
@@ -60,7 +56,7 @@ Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum R
 - **Penamaan tingkat kesulitan kustom dari beatmapset harus mengikuti tema atau pola umum yang berkaitan dengan lagu atau tingkat kesulitannya dan tidak boleh misrepresentatif.** Nama tingkat kesulitan itu misrepresentatif jika menyiratkan tingkat kesulitan yang berbeda (misalnya, menamai tingkat kesulitan "Expert" sebagai "Normal").
 - **Nama tingkat kesulitan tidak boleh hanya terdiri dari satu atau lebih nama pengguna.** Kata-kata yang kebetulan merupakan nama pengguna dapat diterima dalam penamaan tingkat kesulitan selama kata tersebut berkaitan dengan lagu.
 - **Seorang [beatmapset host](/wiki/Beatmap/Beatmap_host) tidak dapat mengindikasikan kepemilikan dalam nama tingkat kesulitan** (misalnya, Beatmapset Host's Insane). Konflik yang disebabkan oleh beatmapping lagu-lagu dengan metadata yang sama dan [tingkat kesulitan kolaboratif](/wiki/Beatmap/Beatmap_collaborations) adalah pengecualian. Namun, [Guest difficulty](/wiki/Beatmap/Guest_difficulty) dapat mengindikasikan kepemilikan dengan nama pengguna atau nama panggilan pembuatnya.
-- **Seorang [beatmapset host](/wiki/Beatmap/Beatmap_host) harus membuat tingkat kesulitan yang jumlahnya sama atau lebih dari pembuat [guest difficulty](/wiki/Beatmap/Guest_difficulty).** hal ini bertujuan untuk memberikan kredit. [Tingkat kesulitan kolaboratif](/wiki/Beatmap/Beatmap_collaborations) hanya dianggap sebagai sebagian dari tingkat kesulitan, dan [drain time](/wiki/Gameplay/Drain_time) akan digunakan untuk menentukan jumlah kontribusi ketika seorang guest telah membuat beatmap yang secara signifikan lebih banyak daripada host.
+- **Seorang [beatmapset host](/wiki/Beatmap/Beatmap_host) harus membuat tingkat kesulitan yang jumlahnya sama atau lebih dari pembuat [guest difficulty](/wiki/Beatmap/Guest_difficulty).** hal ini bertujuan untuk memberi pengakuan. [Tingkat kesulitan kolaboratif](/wiki/Beatmap/Beatmap_collaborations) hanya dianggap sebagai sebagian dari tingkat kesulitan, dan [drain time](/wiki/Gameplay/Drain_time) akan digunakan untuk menentukan jumlah kontribusi ketika seorang guest telah membuat beatmap yang secara signifikan lebih banyak daripada host.
 - **Seorang [beatmapset host](/wiki/Beatmap/Beatmap_host) dan pembuat [guest difficulty](/wiki/Beatmap/Guest_difficulty) dapat mengubah tingkat kesulitan masing-masing sesuai keinginan.** Jika ada ketidaksepakatan antara keduanya, host beatmapset harus menghapus guest difficulty atas permintaan. Jika pembuat guest difficulty tidak dapat dihubungi selama sebulan, mereka akan dianggap setuju dengan perubahan apa pun.
 
```

Comment:
since it been added, add this one.
```- **Beatmapsets tidak boleh berisi bagian-bagian signifikan dari tingkat kesulitan milik pengguna lain tanpa keterlibatan atau izin eksplisit dari mereka.**```